tests/testthat/test-HS_PGA_char.R

```diff
@@ -0,0 +1,50 @@
+test_that("expect_error_missing_values", {
+  expect_error(hs_pga_char(c(1, 2, NA, 4, 5, 6)))
```

Comment:
For `expect_error()`, can you please give the text of the expected error with the `regexp` argument?  That way, we can ensure that not only an error occurs, but also the expected error occurs.  Often, it's easiest to run the command, verify that the expected error occurs, add the text of the error in the `regexp` argument and also set `fixed = TRUE` so that it will not be interpreted as a regular expression, but it will be a substring of the expected error message.

Reply:
Replaced with #13